polars/polars-core/src/named_from.rs

```diff
@@ -144,6 +144,33 @@ impl<T: AsRef<[Series]>> NamedFrom<T, ListType> for Series {
     }
 }
 
+// #[cfg(feature = "dtype-fixed-size-list")]
+// impl<T: AsRef<[Series]>> NamedFrom<T, FixedSizeListType> for Series {
+//     fn new(name: &str, s: T) -> Self {
+//         let series_slice = s.as_ref();
+//         let list_cap = series_slice.len();
+
+//         let dt = series_slice[0].dtype();
+
+//         // inner type is also list so we need the anonymous builder
+//         if let DataType::FixedSizeList(_, _) = dt {
+//             let mut builder = AnonymousListBuilder::new(name, list_cap, Some(dt.clone()));
```

Comment:
Regardless, if we keep this block we'd need to update this line to use the fixed size list's anonymous builder and not the variable size list's builder.

Reply:
Now irrelevant as I moved to the `AnonymousFixedSizeListBuilder`